leftwm-core/src/command.rs

```diff
@@ -25,7 +31,10 @@ pub enum Command {
     FocusWindowTop(bool),
     FocusWorkspaceNext,
     FocusWorkspacePrevious,
-    SendWindowToTag(TagId),
+    SendWindowToTag {
+        window: Option<WindowHandle>,
+        tag: TagId,
+    },
```

Comment:
This isn't exposed to the user yet. When we do it might be worth it having the tag be an option as well, e.g. if no tag is entered we bring the window to the focused tag.